static-assets/components/cstudio-admin/mods/admin-configurations.js

```diff
@@ -168,105 +182,115 @@ CStudioAuthoring.Module.requireModule(
 				);
 
 				// add onchange behavior to display selected
-				itemSelectEl.onchange = function() {
-					var configFilesPath = CStudioAuthoring.Constants.CONFIG_FILES_PATH_ADMIN,
+				this.configInfo.itemSelectEl.onchange = function() {
+          self.loadSelectedConfig();
+				}; // end of change
+      },
+
+      loadSelectedConfig: function() {
+        const self = this,
+              itemSelectEl = this.configInfo.itemSelectEl,
+              editAreaEl = this.configInfo.editAreaEl,
+              editor = this.configInfo.editor,
+              sampleEditor = this.configInfo.sampleEditor;
```

Comment:
Where are these new vars getting used?

Reply:
They used to be sent as parameters to the function (those vars are being used), now they are global properties of the object